tools/build_langserver/langserver/hover_test.go

```diff
@@ -0,0 +1,65 @@
+package langserver
+
+import (
+	"context"
+	"core"
+	"path"
+	"testing"
+	"tools/build_langserver/lsp"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetHoverContent(t *testing.T) {
+	core.FindRepoRoot()
+	ctx := context.Background()
+	filepath := path.Join(core.RepoRoot, "tools/build_langserver/langserver/test_data/example.build")
+	uri := lsp.DocumentURI("file://" + filepath)
+
+	analyzer := newAnalyzer()
+	// Test hovering on the function call
+	content, err := getHoverContent(ctx, analyzer, uri, lsp.Position{Line: 0, Character: 3})
+	assert.Equal(t, nil, err)
+	expected := "def go_library(name:str, srcs:list, asm_srcs:list=None, hdrs:list=None, out:str=None, deps:list=[],\n" +
+		"               visibility:list=None, test_only:bool&testonly=False, complete:bool=True,\n" +
+		"               _needs_transitive_deps=False, _all_srcs=False, cover:bool=True,\n" +
+		"               filter_srcs:bool=True, _link_private:bool=False, _link_extra:bool=True)\n\n" +
```

Comment:
This popup shouldn't include the private arguments (the ones prefixed with underscores), they aren't for users to call.

Reply:
This was directly from the rule definition itself. We would need to reconstruct the definition again if we were to remove the private arguments. Thoughts?